src/components/ScreenSurface.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { Surface } from 'react-native-paper';
+import { createStyles } from './BrandConfigProvider';
+import { useStyles } from '../hooks/useStyles';
+import { ScrollView, View } from 'react-native';
+
+type Props = {
+  children?: React.ReactNode;
+  styles?: ScreenViewStyles;
+} & React.ComponentProps<typeof Surface>;
+
+export function ScreenSurface({
+  children,
+  styles: instanceStyles,
+  ...props
+}: Props) {
+  const { styles } = useStyles(defaultStyles, instanceStyles);
+
+  return (
+    <Surface style={styles.surface} {...props}>
+      <ScrollView overScrollMode="always" showsVerticalScrollIndicator={false}>
```

Comment:
Do we want `overScrollMode` on `android` to be always?

Reply:
I think so?  Not really sure.